spec/views/vms/edit.html.erb_spec.rb

```diff
@@ -3,28 +3,24 @@
 require 'rails_helper'
 
 RSpec.describe 'vms/edit.html.erb', type: :view do
-  let(:vm) do
-    v_sphere_vm_mock 'VM', vm_ware_tools: 'toolsInstalled'
-  end
   let(:configuration) do
-    config = double
-    allow(config).to receive(:description).and_return('hello world')
-    allow(config).to receive(:ip).and_return('127.0.0.1')
-    allow(config).to receive(:dns).and_return('8.8.8.8')
-    config
+    FactoryBot.create(:virtual_machine_config,
+                      name: 'VM',
+                      description: 'hello world',
```

Comment:
Is there a specific reason why the default params of the factory weren't used, i.e. `FactoryBot.create(:virtual_machine_config)`?

Reply:
Don't know, I just refactored this code from it's old state
But good point, I will set this to default